src/ASM/ASMs3DLag.C

```diff
@@ -947,11 +947,14 @@ int ASMs3DLag::findElement (double u, double v, double w,
     return -1;
   }
 
-  const std::array<std::pair<double,int>,3> knot {{
-    {u, svol->basis(0).knotInterval(u)},
-    {v, svol->basis(1).knotInterval(v)},
-    {w, svol->basis(2).knotInterval(w)}
-  }};
+  std::array<std::pair<double,int>,3> knot;
+
```

Comment:
So just remove the blank line 951 and I'm happy, more clear these statements belong together like in the 2D case above.

Reply:
ok